src/components/course/data/service.js

```diff
@@ -50,7 +50,7 @@ export default class CourseService {
     // Check for the course_run_key URL param and remove all other course run data
     // if the given course run key is for an available course run.
     if (this.courseRunKey) {
-      const availableCourseRuns = getAvailableCourseRuns(courseDetails);
+      const availableCourseRuns = getAvailableCourseRuns({ course: courseDetails });
```

Comment:
[inform] Don't worry about the missing coverage on existing code here; all the stuff in `fetchAllCourseData` is getting refactored in the ongoing frontend performance work to migrate these the calls within this function to route loaders and React Query.